omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -514,6 +515,20 @@ def generate_config(node_name)
         PrivateChef["default_listen_address"] = "::"
       end
 
+      # Transition Solr memory and JVM settings from OSC11 to Chef 12.
```

Comment:
One step up: `import_legacy_service_config('old_service_key', 'new_service_key', ['config','keys', 'we', 'care', 'about', 'preserving'])` ?